src/ErrorData.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\ErrorHandler;
+
+use Psr\Http\Message\ResponseInterface;
+
+/**
+ * ErrorData stores content and headers that are suitable for displaying.
+ */
+final class ErrorData
+{
+    /**
+     * @var string The content of error data.
+     */
+    private string $content;
+
+    /**
+     * @var array<string, string|string[]> The headers of error data to add to the response.
+     */
+    private array $headers;
+
+    /**
+     * @param string $content The content of error data.
+     * @param array<string, string|string[]> $headers The headers of error data to add to the response.
+     */
+    public function __construct(string $content, array $headers = [])
+    {
+        $this->content = $content;
+        $this->headers = $headers;
+    }
+
+    /**
+     * Returns a content of error data.
+     *
+     * @return string The content of error data.
+     */
+    public function __toString(): string
```

Comment:
Do we need this method?

Reply:
Of course you do. It is also used in `ErrorHandler`, which returns a string, not a response instance.